prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_OML_20230831-0001.fhir

```diff
@@ -578,58 +578,49 @@
           ]
         },
         "extension": [
```

Comment:
These changes are meant to reflect the new expected structure of an Organization resource. We can alternatively handle this adjustment via sender transform if needed, but I need input from the TI team to determine what makes the most sense. Is this format something that you'd be able to match? Do we need to add a sender transform for TI to move contents from the prior format into the newer format, either as a temporary or long-term solution?

Reply:
Do I understand right that this would mean PID-5.7 would never appear in the outgoing HL7 OML file?

I've been out of the loop for a bit, but the only reason I know of for having 'L' in PID-5.7 for our sample order is because MN's original NBS order sample included it. By contrast, CA's sample does not.

After looking at a number of LOI guides (Natus, MN, TX) -- it's not clear to me if including this will be important or not. For Natus and MN, in the PID segment table they only reference 5.1 and 5.2 ("required" and "required but may be empty", correspondingly). However, table "TABLE. HL70200_US" near the end of the doc calls out use of name type codes in PID-5.7 and NK1-2.7.

Reply:
Curiously, in the Natus guide for NK1, the type code is called out as "required but may be empty".

Reply:
Not quite, this particular change is actually related to how we handle XONs in general, and in particular in this case ORC.21. Basically, ORC.21.2 is now a valueCoding instead of a valueCodeableConcept, and ORC.21.6 and ORC.21.7 are coming from slightly different spots in the FHIR bundle, nested inside the Organization's Identifier. So these changes just move those in the sample FHIR bundle to preserve the output ORC.21.

PID.5 is actually coming from the Patient.name at L883, and PID.5.7 comes from the name.use, with `"use": "official"` corresponding to PID.5.7 of `L`. Since `"use": "official"` was already there, there was no change at least for these messages. There is a slight change there, which is that if there is no `use` specified with the name, formerly we would default to `official` and `L`, but now we don't default to anything and PID.5.7 will be blank. See [here](https://github.com/CDCgov/prime-reportstream/pull/11174/files#diff-e388545ae7c04f2b957d76554f446473310aeb0f2b6175e20b447e1a8f82df86) for where that change in behavior comes from.

Reply:
Sorry -- I think I confused PID-5.7 with PID-6.7. It doesn't look like PID-6 is called out in the Natus LOI at all, so I'm not overly worried if that's impacted.